orc8r/cloud/go/services/certifier/tokens.go

```diff
@@ -0,0 +1,102 @@
+package certifier
+
+import (
+	"crypto/rand"
+	"errors"
+	"fmt"
+	"hash/crc32"
+	"strings"
+
+	"github.com/jxskiss/base62"
+)
+
+type TokenType string
+
+const (
+	Personal TokenType = "op"
```

Comment:
What does "op" stand for? Leave a comment to make it clear, and/or choose something that has semantic meaning (if "op" doesn't)

Reply:
Uhh I intended it to stand for orc8r personal token, though lmk if you have better ideas.